static/app/views/issueDetails/groupSidebar.tsx

```diff
@@ -33,43 +38,32 @@ import {getUtcDateString} from 'sentry/utils/dates';
 import {getAnalyticsDataForGroup} from 'sentry/utils/events';
 import {userDisplayName} from 'sentry/utils/formatters';
 import {isMobilePlatform} from 'sentry/utils/platform';
-import withApi from 'sentry/utils/withApi';
-// eslint-disable-next-line no-restricted-imports
-import withSentryRouter from 'sentry/utils/withSentryRouter';
+import useApi from 'sentry/utils/useApi';
+import {useLocation} from 'sentry/utils/useLocation';
 
-import TagFacets, {
-  BACKEND_TAGS,
-  DEFAULT_TAGS,
-  FRONTEND_TAGS,
-  MOBILE_TAGS,
-  TAGS_FORMATTER,
-} from './tagFacets';
-
-type Props = WithRouterProps & {
-  api: Client;
+type Props = {
   environments: Environment[];
   group: Group;
   organization: Organization;
   project: Project;
   event?: Event;
 };
 
-type State = {
-  allEnvironmentsGroupData?: Group;
-  currentRelease?: CurrentRelease;
-  error?: boolean;
-};
-
-class BaseGroupSidebar extends Component<Props, State> {
-  state: State = {};
-
-  componentDidMount() {
-    this.fetchAllEnvironmentsGroupData();
-    this.fetchCurrentRelease();
-  }
-
-  trackAssign: OnAssignCallback = (type, _assignee, suggestedAssignee) => {
-    const {group, project, organization, location} = this.props;
+export default function GroupSidebar({
```

Comment:
very very small nit, but I am trying to move us off of default exports (very slowly).

Just something to keep in mind of the future.